src/store/widgets/selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { createSelector } from 'reselect';
+import { IStore } from 'store/reducer';
+
+function selectWidgetState({ inverted, location, visible }) {
```

Comment:
Does this selector play any other role than a pure function would? (as far as I understand the purpose of selectors is to cache derived state) Does it make sense to introduce selectors before you suspect any performance issues?

Reply:
no calculations here, so no performance gain - that is true.